test-cases/javascript/state-changing-requests/js-post-event-listener.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body onload="myFunction();">
+<title>CrawlMaze - Testbed for Web Crawlers - form tag</title>
+<h1> POST method from JavaScript EventListener</h1>
+<form id="form">
+  <input type="text" name="test1" value="test">
+  <input type="text" name="test2" value="test">
+  <input type="button" id = 'btn' value="Submit">
+</form>
+<p>
+  The form-data is sent via JavaScript fetch when the button is clicked using an EventListener.
```

Comment:
Nit: Using `an EventListener` is a bit misleading, the other examples are also event listeners. 
This one is registered via `addEventListener` rather than being an inline event listeners as the others. 

Is there any particular reason why you chose to have the function run `onload`, otherwhise we could just run the function synchronously in the inline script which makes it a bit easier to understand.

Currently, it does not seem like the complexity allows to test us crawling specific behavior, but more than happy to talk about that.

Reply:
Thanks, I made that change and will add it in the next PR that includes the test case.